components/api_server/src/routes/report.py

```diff
@@ -114,7 +114,7 @@ def export_report_as_pdf(report_uuid: ReportId):
     renderer_host = os.environ.get("RENDERER_HOST", "renderer")
     renderer_port = os.environ.get("RENDERER_PORT", "9000")
     render_url = f"http://{renderer_host}:{renderer_port}/api/render"
-    query_string = f"?{bottle.request.query_string}" if bottle.request.query_string else ""
+    query_string = "?hide_toasts=true" + (f"&{bottle.request.query_string}" if bottle.request.query_string else "")
```

Comment:
From code perspective, I don't really understand why there is `hide_toasts`. Perhaps something for a comment or otherwise in the developer documentation?

Reply:
Added a comment:

```suggestion
    # Tell the frontend to not display toast messages to prevent them from being included in the PDF:
    query_string = "?hide_toasts=true" + (f"&{bottle.request.query_string}" if bottle.request.query_string else "")
```